docs/adr/2022-12-01_001-record-architectural-decisions.md

```diff
@@ -0,0 +1,30 @@
+---
+slug: 1
+title: | 
+  1. Record Architecture Decisions
+authors: []
+tags: [Proposed]
+---
+
+## Status
+
+Proposed
+
+## Context
+
+We are in search for a means to describe our technical architecture.
+
+We prefer light-weight documentation methods which accomodate change easily.
+
+The [Hydra team](https://github.com/input-output-hk/hydra)
+has had success with the solution described below.
+See the [Hydra ADR 1](https://github.com/input-output-hk/hydra/blob/master/docs/adr/2021-06-05_001-record-architectural-decisions.md).
+
+## Decision
+
+* We will use _Architecture Decision Records_, as described by Michael Nygard in this [article](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions).
+* We will follow the convention of storing those ADRs as Markdown formatted documents stored under `docs/adr` directory, as exemplified in Nat Pryce's [adr-tools](https://github.com/npryce/adr-tools). This does not imply we will be using `adr-tools` itself.
+
+## Consequences
```

Comment:
I just realized that I misinterpreted this section. It's the immediate consequences, not to be confused with the "experience report" that can come after. I'll ~steal~ copy the prose from the Hydra ADR 1 :grin: .